packages/cli/internal/pkg/cli/workflow/workflow_tasks_test.go

```diff
@@ -200,6 +201,58 @@ func (s *GetWorkflowTasksTestSuite) TestGetWorkflowTasks_WithTaskNameFailure() {
 	}
 }
 
+func (s *GetWorkflowTasksTestSuite) TestGetWorkflowTasks_WithExitCodeUnknown() {
+	defer s.ctrl.Finish()
+	s.mockProjectClient.EXPECT().Read().Return(s.testProjSpec, nil)
+	s.mockConfigClient.EXPECT().GetUserId().Return(testUserId, nil)
+	s.mockDdb.EXPECT().GetWorkflowInstanceById(ctx.Background(), testProjectName, testUserId, testRunId).Return(ddb.WorkflowInstance{ContextName: testContext1Name}, nil)
+	s.mockCfn.EXPECT().GetStackInfo(testContext1Stack).Return(cfn.StackInfo{Outputs: map[string]string{"WesUrl": testWes1Url}}, nil)
+	s.mockWes.EXPECT().GetRunLog(ctx.Background(), testRunId).Return(wes_client.RunLog{
+		RunId: testRunId,
+		TaskLogs: []wes_client.Log{{
+			Name:      testTaskCompositeName,
+			StartTime: testStartTime.UTC().Format("2006-01-02T15:04:05Z"),
+			EndTime:   testStopTime.UTC().Format("2006-01-02T15:04:05Z"),
+			ExitCode:  nil,
+		}},
+	}, nil)
+
+	tasks, err := s.manager.GetWorkflowTasks(testRunId)
+	if s.Assert().NoError(err) {
+		s.Assert().Equal(testTaskName, tasks[0].Name)
+		s.Assert().Equal(testTaskJobId, tasks[0].JobId)
+		s.Assert().True(tasks[0].StartTime.Equal(testStartTime.Truncate(time.Second)))
+		s.Assert().True(tasks[0].StopTime.Equal(testStopTime.Truncate(time.Second)))
+		s.Assert().Empty(nil, "NA")
```

Comment:
This tests if `nil` is "empty" which it always is. Perhaps you meant to test that `ExitCode` is empty?

Reply:
awesome catch, thanks Mark! i've updated the test.